### PR TITLE
chore: added API EXPLORER tests

### PR DESCRIPTION
**Pull request by sanjam-deriv**

### CODE REVIEW

File: src/features/Apiexplorer/__tests__/ApiExplorer.test.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+  it('should render the dropdown', () => {
+    const dropdown = screen.getByTestId('dropdown');
+    expect(dropdown).toBeInTheDocument();
+  });
+
+  it('should render the textarea', () => {
+    const textarea = screen.getByPlaceholderText('Request JSON');
+    expect(textarea).toBeInTheDocument();
+  });
```
[the remark this answers is not in the document]
Comment (sanjam-deriv) [reply src/features/Apiexplorer/__tests__/ApiExplorer.test.tsx:44]: These tests are already included in the indiviual components and hooks

[the remark this answers is not in the document]
Comment (hubert-deriv) [reply src/features/Apiexplorer/__tests__/ApiExplorer.test.tsx:44]: If that is the case, then it's good I think.

**Review by hubert-deriv — APPROVED** [review]

File: src/features/Apiexplorer/__tests__/ApiExplorer.test.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  useHistory: () => ({
+    push: jest.fn(),
+  }),
+}));
+
+jest.mock('@site/src/hooks/useAuthContext');
+
+const mockUseAuthContext = useAuthContext as jest.MockedFunction<() => Partial<IAuthContext>>;
+
+describe('ApiExplorerFeatures', () => {
```
Comment (mohsen-deriv) [inline src/features/Apiexplorer/__tests__/ApiExplorer.test.tsx:22]: please add these test cases as well: 

- what if user is logged out? 
- test case for pressing the button and websocket messages, api calls ,etc. 
- test case for changing the value of the dropdown
- test case for changing the payload and pressing the button

**Review by mohsen-deriv — CHANGES_REQUESTED** [review]

File: src/features/Apiexplorer/__tests__/ApiExplorer.test.tsx (hunk 2 above)
Comment (sanjam-deriv) [reply src/features/Apiexplorer/__tests__/ApiExplorer.test.tsx:22]: These test cases arfe laready being writen for the indiviual components I don't think 
why do we need them for the wrapper as well

**Review by mohsen-deriv — APPROVED** [review]

**Pull request merged**